Tasks/IISWebAppManagementOnMachineGroup/README.md

```diff
@@ -1,2 +1,76 @@
- 
- 
\ No newline at end of file
+# IIS Web Application Management
+
+## Overview
+
+This task is used to manage an IIS web site or a web application or a virtual directory or an application pool. You can Create or update a Web Application or a Virtual Directory as a child of an existing IIS Web Site. 
```

Comment:
To many "or" I think it can be worded saying manage website, web application, virtual directory, or application pool.